frame/contracts/src/chain_extension.rs

```diff
@@ -29,6 +29,20 @@
 //! required for this endeavour are defined or re-exported in this module. There is an
 //! implementation on `()` which can be used to signal that no chain extension is available.
 //!
+//! # Using multiple chain extensions
+//!
+//! Often there is a need for having multiple chain extensions. This is often the case when
+//! some generally useful off-the-shelf extensions should be included. To have multiple chain
+//! extensions they can be put into a tuple which is then passed to `[Config::ChainExtension]` like
+//! this `type Extensions = (ExtensionA, ExtensionB)`.
+//!
+//! However, only extensions implementing [`RegisteredChainExtension`] can be put into a tuple.
+//! This is because the [`RegisteredChainExtension::ID`] is used to decide which of those extensions
+//! should should be used when the contract calls a chain extensions. Extensions which are generally
+//! useful should claim their `ID` with [the registry](https://github.com/paritytech/chainextension-registry)
+//! so that no collisions with other vendors will occure. **Chain specific extensions must use the
+//! reserved `ID = 0` that can't be registered with the registry.**
```

Comment:
```suggestion
//! so that no collisions with other vendors will occur.
//! 
//! **Chain specific extensions must use the reserved `ID = 0` that they can't be registered with the registry.**
```

Reply:
The last sentence doesn't make sense to me "that they can't be registered ...". Is this missing a word?

Reply:
Ok updated your comment.